scripts/test-dev.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+# TEST_SUBDIR="general-tests"
+# test_name="declare-v2-test"
+
+TEST_SUBDIR_PENDING="true"
+TEST_NAME_PENDING="true"
+export STARKNET_HARDHAT_DEV=1
```

Comment:
I would appreciate a comment of what this variable does? The name seems generic or misleading.